ui/imports/shared/stores/CurrenciesStore.qml

```diff
@@ -1,11 +1,16 @@
 import QtQuick 2.13
 import utils 1.0
+import "../../../app/AppLayouts/Profile/stores"
 
 QtObject {
     id: root
 
     property var locale: Qt.locale(localAppSettings.language)
```

Comment:
This is wrong... `localAppSettings.language` has the _translation_ language, not the locale!

Reply:
As I said above, just remove it

Reply:
`Qt.locale()` will always give you the correct one

Reply:
Hmm interesting... Do we always want to use the system locale regardless of the app language chosen by the user (in case they don't match)? It seemed to me a user might want both things to go hand in hand (like, display the app in Spanish with the numbers formatted the Spanish way)

Reply:
Definitely; we don't even have own translations for every language out there